tests/backend/user/test_user_claim.py

```diff
@@ -60,7 +75,7 @@
                 user_id=mallory_invitation.user_id,
             )
 
-    assert prep[0] == {"status": "ok"}
+    assert prep[0]["status"] == "ok" and prep[0]["user_certificate"] == b"<foo>"
```

Comment:
Il vaut mieux mettre deux asserts plutôt qu'un `and`, histoire de simplifier le travail de pytest qui va réécrire  l'ast des assertions pour avoir un message d'erreur de qualité (cf. https://docs.pytest.org/en/3.0.5/assert.html#advanced-assertion-introspection)